domains/game/Game.jsx

```diff
@@ -0,0 +1,187 @@
+                </div>
+                <div class="row">
+                    <div class="cell">
+                        <div id="cell40"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell41"></div>
+                    </div>
+                    <div class="cell obstacle">
+                        <div id="cell42"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell43"></div>
+                    </div>
+                    <div class="cell obstacle">
+                        <div id="cell44"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell45"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell46"></div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="cell">
+                        <div id="cell50"></div>
+                    </div>
+                    <div class="cell obstacle">
+                        <div id="cell51"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell52"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell53"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell54"></div>
+                    </div>
+                    <div class="cell obstacle">
+                        <div id="cell55"></div>
+                    </div>
+                    <div class="cell bottom-exit">
+                        <div id="cell56"></div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="cell">
+                        <div id="cell60"></div>
+                    </div>
+                    <div class="cell right-exit">
+                        <div id="cell61"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell62"></div>
+                    </div>
+                    <div class="cell obstacle">
+                        <div id="cell63"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell64"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell65"></div>
+                    </div>
+                    <div class="cell">
+                        <div id="cell66"></div>
+                    </div>
+                </div>
+                <button name="action" type="submit" value="east" class="tilt">
+                    Tilt
+                </button>
+            </div> */}
+        </View>
+    );
+};
+
+Game.propTypes = {
+    gameId: PropTypes.number,
+    playerName: PropTypes.string,
```

Comment:
Add isRequired to the propTypes when it is mandatory. Otherwise you get no warning when it is forgotten